File: week1_py/day4/DailyChallenge/exercise2.py
#Write a program that asks a string to the user, 
# and display a new string with any duplicate 
# consecutive letters removed.
#F/inal strings won’t include words with double 
# letters (e.g. “passing”, “lottery”).
text = input('enter your string:\n')
answer = ''
for i,letter in enumerate(text):
    if not answer or answer[-1] != letter:
        answer += letter
print(answer)

File: week9/day4/ExercisesXPGold#2/exercise2.js
//Write a JavaScript program to remove duplicate items in an array.
function removeDuplicates(array){
    return Array.from(new Set(array))
}
console.log('[1, 2, 3 ]', removeDuplicates([1, 1, 2, 3, 3, 3]))
console.log('[] ', removeDuplicates([]))
console.log('[true, false] ', removeDuplicates([true, true, false, false]))
console.log('["a", "b", "a", "b", "c"] ', removeDuplicates(["a", "b", "c"]))